### PR TITLE
third party send-materials fix

### PR DESCRIPTION
**Pull request by rburema**
Updated or new materials wheren't always sent to the (UM) printer, causing the name to be 'unkown'.

### CODE REVIEW

**Comment by LipuFei** [comment]
Together with https://github.com/Ultimaker/Uranium/pull/439

**Comment by ChrisTerBeke** [comment]
The tests fail now

File: plugins/UM3NetworkPrinting/src/SendMaterialJob.py [hunk 1]
```diff
@@ -91,22 +86,22 @@ def _determineMaterialsToSend(local_materials: Dict[str, LocalMaterial],
...
+                # If the material does not have to be sent we skip it.
                 continue
 
-            file_name = os.path.basename(file_path)
-            material_id = urllib.parse.unquote_plus(mime_type.stripExtension(file_name))
-            if material_id not in materials_to_send:
-                # If the material does not have to be sent we skip it.
+            file_path = container_registry.getContainerFilePathById(root_material_id)
+            if not file_path:
+                Logger.log("e", "Cannot get file path for material container [%s]", root_material_id)
```
Comment (nallath) [inline plugins/UM3NetworkPrinting/src/SendMaterialJob.py]: nitpick; Isn't this a warning?

Comment (LipuFei) [reply plugins/UM3NetworkPrinting/src/SendMaterialJob.py]: Changed to warning in ee7228461

**Pull request merged**